src/View/Helper/FormHelper.php

```diff
@@ -1722,7 +1722,11 @@ public function submit($caption = null, array $options = [])
         if (!is_string($caption) && empty($caption)) {
             $caption = __d('cake', 'Submit');
         }
-        $options += ['type' => 'submit', 'secure' => false];
+        $options += [
+            'type' => 'submit',
+            'secure' => false,
+            'templateVars' => []
```

Comment:
Option added. Maybe it is worth adding the options to the others method using it? It is currently only documented in the Cookbook.